src/openrct2/paint/tile_element/Path.cpp

```diff
@@ -695,6 +701,11 @@ void path_paint(paint_session * session, uint8 direction, uint16 height, rct_til
         }
     }
 
+    if (gCurrentViewportFlags & VIEWPORT_FLAG_HIGHLIGHT_PATH_ISSUES)
+    {
+        imageFlags = 0x21700000;
```

Comment:
Split this up into remap type and shifted colours/filters

Reply:
NB: the same magic value is used in other places, when selecting scenery to save with a track design. [Fence.cpp line 170](https://github.com/OpenRCT2/OpenRCT2/pull/6996/files#diff-7f10c704c1b6b6a80b7bedad11219eb2R170) in particular stands out as it shows up in this PR.